src/bootloader_env/uboot_env.rs

```diff
@@ -0,0 +1,46 @@
+use anyhow::{bail, ensure, Context, Result};
+use std::process::Command;
+
+pub fn bootloader_env(key: &str) -> Result<String> {
+    let value = Command::new("sudo").arg("fw_printenv").arg(key).output()?;
+    if !value.status.success() {
+        bail!("fw_printenv {} failed", key);
```

Comment:
just an info. I try to  format text always inline (if possible):
`bail!("fw_printenv {key} failed");`

Reply:
done